twill-api/src/main/java/org/apache/twill/api/TwillPreparer.java

```diff
@@ -227,15 +227,52 @@
   TwillPreparer addSecureStore(SecureStore secureStore);
 
   /**
-   * Set the log level for Twill applications running in a container.
+   * Set the root log level for Twill applications in all containers.
    *
    * @param logLevel the {@link LogEntry.Level} that should be set.
    *                 The level match the {@code Logback} levels.
    * @return This {@link TwillPreparer}.
+   * @deprecated Use {@link #setRootLogLevel(LogEntry.Level)} instead.
    */
+  @Deprecated
   TwillPreparer setLogLevel(LogEntry.Level logLevel);
 
   /**
+   * Set the root log level for Twill applications in all containers.
+   *
+   * @param logLevel the {@link LogEntry.Level} that should be set.
+   *                 The level match the {@code Logback} levels.
+   * @return This {@link TwillPreparer}.
+   */
+  TwillPreparer setRootLogLevel(LogEntry.Level logLevel);
+
+  /**
+   * Set the root log level for a particular runnable
+   *
+   * @param runnableName The name of the runnable to set the log level.
+   * @param logLevel the {@link LogEntry.Level} that should be set.
+   * @return This {@link TwillPreparer}.
+   */
+  TwillPreparer setRootLogLevel(String runnableName, LogEntry.Level logLevel);
```

Comment:
I prefer not to introduce the `setRootLogLevel` and the deprecated `setLogLevel` replaced with call to `setLogLevel(Map)` with logger name `Logger.ROOT_LOGGER_NAME` to set root logger log level.